lib.js

```diff
@@ -93,3 +93,22 @@ export const generateTestsuiteResult = (suiteResult) => {
 
   return result.join(EOL);
 };
+
+export const generateTestsuiteLogs = (suiteResult) => {
+  let result = [];
+  const testscases = suiteResult.testcase;
+
+  if (testscases) {
+    let errLogs = [...testscases.map(testcase => testcase["system-err"] ? testcase["system-err"].join(EOL).split(EOL).join(EOL + '\t') : "")].filter(a => a != "");
+    if (errLogs.length) {
+      result.push(EOL, "   > Error Log output:", EOL, EOL, '\t', ...errLogs)
+    }
+
+    let outLogs = [...testscases.map(testcase => testcase["system-out"] ? testcase["system-out"].join(EOL).split(EOL).join(EOL + '\t') : "" )].filter(a => a != "")
+    if (outLogs.length) {
+      result.push(EOL, "   > Standard Log output:", EOL, EOL, '\t', ...outLogs)
+    }
+  }
+
+  return result.join('');
```

Comment:
This can be joined by EOL instead of pushing EOL to result array above for errLogs and outLogs. In that way if a case of either of one is present also the formatting would always be separated by equal line breaks.

```suggestion
  return result.join(EOL);
```

Reply:
I couldn't get all the log lines to line up correctly any other way. Would appriciate any help you're able to give here.